demo/main.go

```diff
@@ -138,17 +136,22 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
+	fmt.Println("[DEBUG] Starting resharing")
```

Comment:
the convention in the demo project seems to be to add `[+]` or `[-]` at the start of logs (whether they're expected or unexpected) to clearly separate them from internal logger logs